code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -47,3 +47,4 @@
 	new /obj/item/clothing/mask/gas(src)
 	new /obj/item/device/flash(src)
 	new /obj/item/weapon/switchtool/holo(src)
+	new /obj/item/clothing/shoes/kneesocks/pinkknee(src)
```

Comment:
Why not just have the subtype as kneesocks/pink?

pink knee sounds like an ailment